diffengine/__init__.py

```diff
@@ -358,7 +360,7 @@ def load_config(prompt=True):
     global config
     config_file = os.path.join(home, "config.yaml")
     if os.path.isfile(config_file):
-        config = yaml.load(open(config_file), Loader=yaml.FullLoader)
+        config = EnvYAML(config_file)
```

Comment:
Here is the actual change.

All the others one are automatic whitespaces replacements that my IDEs does automatically